### DISCUSSION TITLE
Graph: Stacked Bar inside rounded corners

### DISCUSSION THREAD

**Discussion by Flowko** [f5/unovis#573] [Ideas]
currently stacked bars only support top bar rounded corners, would love to have the ability to set rounded corners for inside bars as well, to get something like this:

![image](https://github.com/f5/unovis/assets/35883748/08246898-95ac-49c9-ac1a-434ba423c1ea)

**Comment by rokotyan** [comment]
Thanks for the feedback, we'll think about it.

**Comment by tgb20** [comment]
This is also something I would be interested in

**Comment by ManuelOchagavia** [comment]
I think this also would fix this kind of behaviour, doesn't it? (Only rounded corners on the top bar)

![image](https://github.com/user-attachments/assets/68d9bb55-a1d7-4338-b3e4-6cf94d872f53)

**Comment by VictorPruefer** [comment]
> I think this also would fix this kind of behaviour, doesn't it? (Only rounded corners on the top bar)
> 
> ![image](https://github.com/user-attachments/assets/68d9bb55-a1d7-4338-b3e4-6cf94d872f53)

Would also appreciate seeing this implemented. @ManuelOchagavia Have you found any workaround for this issue?

**Comment by lee00678** [comment]
I'm converting this into a discussion. The team gave it some thought, by simply adding rounded corner, there will be gaps in between bars. Doesn't look good. But if we make the top bars bigger, is that true to the data? 

Would love to hear some ideas/feedbacks! 
![Screenshot 2025-05-23 at 9 23 19 AM](https://github.com/user-attachments/assets/9937c1d8-7767-40ee-bc68-1593f0857989)
